docs/Installation.md

```diff
@@ -361,8 +361,8 @@ install -v -m 755 -d /usr/local/etc/zonemaster
 install -v -m 640 -g zonemaster ./backend_config.ini /usr/local/etc/zonemaster/
 install -v -m 775 -g zonemaster -d /var/log/zonemaster
 install -v -m 775 -g zonemaster -d /var/run/zonemaster
-install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
-install -v -m 755 ./zm_testagent-bsd /usr/local/etc/rc.d/zm_testagent
+install -v -m 755 ./zm-rpcapi.bsd /usr/local/etc/rc.d/zm-rpcapi
+install -v -m 755 ./zm-testagent.bsd /usr/local/etc/rc.d/zm-testagent
 ```

```

Comment:
I don't see why this should land in the Installation document. This is something for the Upgrade document.

Reply:
I you miss it, it will create problem. If you have an older version, you are directed to the installation document. If you do not save the database, you might take the easier step and remove and reinstall. We could add the following two commands to the command set:
```
rm -f /usr/local/etc/rc.d/zm_rpcapi
rm -f /usr/local/etc/rc.d/zm_testagent
```

That is shorter and easier than text.